### ISSUE TITLE
Bug: File column always resets to far right position.

### ISSUE THREAD

**Issue by that0n3guy** [RafaelGB/obsidian-db-folder#18]
**Describe the bug**
The "file" column does not maintain its order and always becomes the farthest right column

**To Reproduce**
1. go to you database file
2. drag columns around until the "file" column in to the left (in the middle or farthest left)
3. switch files
4. switch back to database file
5. The "file" column will have relocated back to the farthest right column

**Expected behavior**
I expect the columns to stay in the order I drag-n-dropped them.

![Obsidian_h7q04Aqoyn](https://user-images.githubusercontent.com/113870/165979552-6339c5c1-55d7-46ed-a82d-8b01a1090187.gif)

**Desktop (please complete the following information):**
 - OS: Windows 11
  - Version: Obsidian 0.14.7
  - dbfolder version: 0.0.7

**Comment by RafaelGB** [comment]
Actually file column is a metadata internal type, not support yet persist the order.   I will try to persist it with future versions

**Comment by RafaelGB** [comment]
after 0.1.1 that resolves some bugs, I am starting with this one

**Comment by RafaelGB** [comment]
was fast! Now this feature is avaliable with 0.1.2

**Comment by that0n3guy** [comment]
It didn't work at first...  I was dragging the other columns to the right of the "file" column then switching files and coming back... the file column would reset to the far right.  (just like in my first post)

BUT, once I grabbed the file column and dragged it, it now remembers where it should be, even if I drag the others.

I think its a bug, just I can't reproduce it now that I've dragged the file column.
